pkg/minikube/cluster/cluster.go

```diff
@@ -113,6 +114,25 @@ func StartHost(api libmachine.API, config cfg.MachineConfig) (*host.Host, error)
 	s, err := h.Driver.GetState()
```

Comment:
we could make our own ErrorType for not found and then make all drivers return that so we can use it as a generic machine not found across all drivers

Reply:
I added new function isMachineExists.
This func checks all driver's ErrorMachineNotExist.